app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js

```diff
@@ -28,6 +28,9 @@ RedirectWithSearchAndHash.propTypes = {
 
 const RedirectWithLocation = withRouter(RedirectWithSearchAndHash)
 
+const CLUSTERS_PLOT = `clusters`
+const METADATA_PLOT = `metatdata`
```

Comment:
I guess it is a typo.
```suggestion
const METADATA_PLOT = `metadata`
```